resource-aggregate/cqrs/eventbus/nats/configV2.go

```diff
@@ -0,0 +1,21 @@
+package nats
+
+import (
+	"fmt"
+
+	nats "github.com/nats-io/nats.go"
+	"github.com/plgd-dev/cloud/pkg/security/certManager/client"
+)
+
+type ConfigV2 struct {
```

Comment:
why is not v2 part of the namespace / import but part of the struct?

Reply:
This is tmp state for now. After we migrate al services we will replace Config with Config2.

Reply:
Why do we need Config2 and cannot stay on Config? There is no backwards compatibility with plgd cloud1. You can rename it right away to Config.

Reply:
It will be renamed in RD merge request.